### PR TITLE
feat: LT-995 - Withdraw active application

### PR DESCRIPTION
**Pull request by jfaracik**
image

* Added view for withdrawing applications
* Added button to 'application' detail view to access withdraw page
* Changed some table headers to use content system
* Added deprecation notice to old `get_string`

* E2E not written yet - waiting on LT-1519 to be merged in.

### CODE REVIEW

File: templates/applications/application.html [hunk 1]
```diff
@@ -10,37 +10,47 @@
...
+            <div class="govuk-grid-column-one-quarter">
+                <p class="lite-heading-s">{% lcs 'APPLICATION_EXPORT_TYPE' %}</p>
                 <p class="govuk-label">
                     {{ application.export_type.value }}
                 </p>
             </div>
-            <div class="govuk-grid-column-one-third">
-                <p class="lite-heading-s">Last Updated</p>
+            <div class="govuk-grid-column-one-quarter">
+                <p class="lite-heading-s">Status</p>
```
Comment (efahy) [inline templates/applications/application.html]: lcs!

**Comment by efahy** [comment]
Are there any UI unit tests you could/should add?

File: ui_automation_tests/features/withdraw_application.feature [hunk 2]
```diff
@@ -0,0 +1,14 @@
+Feature: I want to be able to withdraw an active application
+
+  @LT-995-Withdraw-active-application
```
Comment (phlptt) [inline ui_automation_tests/features/withdraw_application.feature]: underscores

File: ui_automation_tests/conftest.py [hunk 3]
```diff
@@ -483,6 +478,12 @@ def i_choose_to_make_minor_edits(driver):
     application_edit_type_page.click_change_application_button()
 
 
+@when("I click continue")  # noqa
+@then("I click submit")  # noqa
```
Comment (phlptt) [inline ui_automation_tests/conftest.py]: I click submit is a when not then

File: ui_automation_tests/pages/application_page.py [hunk 4]
```diff
@@ -1,12 +1,17 @@
...
-        self.ecju_query_response_text = 'Respond to query'  # text
-        self.ecju_queries_closed = "closed-ecju-query"  # ID
-        self.edit_application_link = "a[href*='/edit-type/']"
-        self.edit_application_button = "edit-application-button"  # ID
+
+class ApplicationPage(BasePage):
+    BUTTON_WITHDRAW_APPLICATION_ID = "button-withdraw-application"
+    BUTTON_EDIT_APPLICATION_ID = "button-edit-application"
+    LABEL_APPLICATION_STATUS_ID = "label-application-status"
+    ecju_query_tab = "ecju-queries-tab"  # ID
```
Comment (phlptt) [inline ui_automation_tests/pages/application_page.py]: different naming variable styles

**Pull request merged**